src/overlay/overlay.scss

```diff
@@ -0,0 +1,184 @@
+.Overlay-hidden {
+  display: none !important;
+}
+
+.Overlay {
+  display: flex;
+  min-width: 320px;
+  max-width: calc(100vw - 64px);
+  flex-direction: column;
+  background-color: var(--color-canvas-default);
+  // stylelint-disable-next-line primer/borders
+  border-radius: 12px;
+  box-shadow: var(--color-shadow-extra-large);
```

Comment:
Let's use the overlay-specific shadows for now (while this isn't refactored in primer primitives :D

https://github.com/primer/primitives/blob/1bcb44f51bbf612302e9204cf4e1dc187ec6be94/data/colors/vars/component_light.ts#L22-L25